src/main/java/fr/xephi/authme/service/yaml/YamlFileResourceProvider.java

```diff
@@ -0,0 +1,30 @@
+package fr.xephi.authme.service.yaml;
+
+import ch.jalu.configme.resource.YamlFileResource;
+import org.yaml.snakeyaml.parser.ParserException;
+
+import java.io.File;
+
+/**
+ * Extension of {@link YamlFileResource}, which throws a specific exception if a file cannot be loaded.
```

Comment:
Oops this is no longer an extension.
Initially I wanted to extend ConfigMe's YamlFileResource class but due to great design (cough, cough) it proved troublesome to wrap the initializing method there into its own exception (I don't have access to the file anymore, saving the file in an individual field also doesn't work because the exception will already have been thrown before the file field is set...).
I'll adjust the comment.